src/app/Main/Body/QueryPage/Results/TableResults/TableContent-Header.component.tsx

```diff
@@ -66,14 +67,15 @@ export function TableContentHeaderComponent(props: {
         </StyledTableCell>
     );
     props.headerValues.forEach((headerValue): void => {
-        let headerTitle = headerValue;
-        if (props.tableOption === "relative") {
-            if (props.isCol) {
-                headerTitle = `${headerValue} ${t("Results.Unit")}`;
-            } else {
-                headerTitle = `${t("Results.TableHeadRelative")}`;
-            }
+        let headerTitle = t(`FilterValues.${props.colAttribute}.${headerValue}`) ;
+        if (props.tableOption === "relative" && props.isCol) {
+            headerTitle = `${headerTitle} ${t("Results.Unit")}`;
+        } else if (props.tableOption === "relative" && !props.isCol) {
+            headerTitle = t("Results.TableHeadRelative");
+        } else if (props.tableOption === "absolute" && !props.isCol) {
+            headerTitle = t("Results.TableHead");

```

Comment:
Suggestion to make the code more understandable: The main difference here is if the header is a data header or not, so i would differentiate this first and then branch for the data type:
```
let headerTitle = '';
if(isCol) {
  headerTitle = t(...);
  if(relative) {
    headerTitle = headerTitle + t(...)
  }
else {
  if(relative) t(...)
  else if(absolute) t(...)
}
```

Reply:
Done. Change if-else